src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/hmctsresponsereviewed/HmctsResponseReviewedSubmittedHandler.java

```diff
@@ -53,24 +57,40 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
 
         if (sscsCaseData.getIsInterlocRequired() != null && sscsCaseData.getIsInterlocRequired().equals("Yes")) {
             String whoToReview = sscsCaseData.getSelectWhoReviewsCase().getValue().getCode().equals("reviewByJudge") ? "Judge" : "TCW";
-            updateCase(sscsCaseData, callback.getCaseDetails().getId(), EventType.VALID_SEND_TO_INTERLOC, "Send to interloc", "Send a case to a " + whoToReview + " for review");
+            updateCase(
```

Comment:
I think this call first is in effect just a triggerEventV2

Reply:
Yeah it should be. Had to keep it generic since its a common code - updateCase .

Reply:
Actually, there's no real reason for it to be generic. It might be better to split out the 2 types of calls into their own methods. Might be a bit clearer for devs in the long-term